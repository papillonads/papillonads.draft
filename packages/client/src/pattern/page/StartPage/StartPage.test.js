import React from 'react'
import startPageDefault from './StartPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<StartPage />', () => {
  describe('Snaphot', () => {
    test('must match startPageDefault', () => {
      expect(global.renderToJSON(startPageDefault)).toMatchSnapshot()
    })
  })
})
