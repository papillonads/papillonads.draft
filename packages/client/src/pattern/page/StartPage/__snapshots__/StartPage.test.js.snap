// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartPage /> Snaphot must match startPageDefault 1`] = `<div />`;
